crates/gitql-ast/src/function.rs

```diff
@@ -24,6 +24,8 @@ lazy_static! {
         map.insert("ltrim", text_left_trim);
         map.insert("rtrim", text_right_trim);
         map.insert("len", text_len);
+        map.insert("ascii", text_ascii);
```

Comment:
This PR is related to `char` function so please remove unrelated `ascii` implementation